specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2020-06-01/imds.json

```diff
@@ -0,0 +1,884 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/IdentityErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Identity Info for the VM": {
+            "$ref": "./examples/GetIdentityInfo.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Compute": {
+      "type": "object",
+      "properties": {
+        "azEnvironment": {
+          "type": "string",
+          "description": "This is the name of the environment in which the VM is running."
+        },
+        "isHostCompatibilityLayerVm": {
+          "type": "string",
+          "description": "Identifies if the VM runs on the Host Compatibility Layer."
+        },
+        "location": {
+          "type": "string",
+          "description": "This is the Azure Region in which the VM is running."
+        },
+        "name": {
+          "type": "string",
+          "description": "This is the name of the VM."
+        },
+        "offer": {
+          "type": "string",
+          "description": "This is the offer information for the VM image. This value is only present for images deployed from the Azure Image Gallery."
+        },
+        "osType": {
+          "type": "string",
+          "description": "This value indicates the type of OS the VM is running, either Linux or Windows."
+        },
+        "placementGroupId": {
+          "type": "string",
+          "description": "This is the placement group of your Virtual Machine Scale Set."
+        },
+        "plan": {
+          "type": "object",
+          "description": "This contains the data about the plan.",
+          "$ref": "#/definitions/PlanProperties"
+        },
+        "publicKeys": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PublicKeysProperties"
+          },
+          "description": "This is information about the SSH certificate"
+        },
+        "platformFaultDomain": {
+          "type": "string",
+          "description": "This is the fault domain in which the VM."
+        },
+        "platformUpdateDomain": {
+          "type": "string",
+          "description": "This is the update domain in which the VM."
+        },
+        "provider": {
+          "type": "string",
+          "description": "This is the provider of the VM."
+        },
+        "publisher": {
+          "type": "string",
+          "description": "This is the publisher of the VM image."
+        },
+        "resourceGroupName": {
+          "type": "string",
+          "description": "This is the resource group for the VM."
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "This is the fully qualified ID for the VM."
+        },
+        "securityProfile": {
+          "type": "object",
```

Comment:
This line should be removed since it will be overridden by whatever the `$ref` refers to.